### PR TITLE
[local] added warning for not supporting aws logging

### PR DESCRIPTION
**Pull request by iamhopaul123**
Issue #, if available:
#800 

Description of changes:
Warn if aws logging is set in task definition.

----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**  
- [x] Unit tests passed
- [x] Integration tests passed
- [ ] Unit tests added for new functionality
- [ ] Listed manual checks and their outputs in the comments ([example](https://github.com/aws/amazon-ecs-cli/pull/750#issuecomment-472623042))
- [ ] Link to issue or PR for the integration tests: 

**Documentation**  
- [ ] Contacted our doc writer
- [ ] Updated our README
----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by iamhopaul123** [comment]
### Manual Test

#### 1. With AWS logging 

`ecs-cli local up -r arn:aws:ecs:us-west-2:403971813171:task-definition/e2e-fargate-test-service:1`

```
INFO[0000] Reading task definition from e2e-fargate-test-service:1
 
INFO[0000] Task Definition network mode is ignored when running containers locally. Tasks will be run in the ecs-local-network.  networkMode=awsvpc
INFO[0000] PID mode can only be set to 'host' when running tasks locally.  pid=
INFO[0000] IPC mode can only be set to 'host' or 'none' when running tasks locally.  ipc=
WARN[0000] AWS log will not be supported                
INFO[0000] Creating network: ecs-local-network...       
INFO[0015] Created network ecs-local-network with ID 84649ed4e092b06c783a4e97f0cb1d2462bf2eaae7586b6a42bb1cef52fbc959 
INFO[0015] Created the amazon-ecs-local-container-endpoints container with ID 67a5669eada760ca766dfde7b90e93c1695e94412691fb420470a3618072b6b1 
INFO[0016] Started container with ID 67a5669eada760ca766dfde7b90e93c1695e94412691fb420470a3618072b6b1 
INFO[0016] Using docker-compose.ecs-local.yml to start containers 
```

#### 2. Without AWS logging

`ecs-cli local up -r arn:aws:ecs:us-west-2:403971813171:task-definition/e2e-ec2-tutori
al-taskdef:2`

```
INFO[0000] Reading task definition from e2e-ec2-tutorial-taskdef:2
 
INFO[0000] PID mode can only be set to 'host' when running tasks locally.  pid=
INFO[0000] IPC mode can only be set to 'host' or 'none' when running tasks locally.  ipc=
docker-compose.ecs-local.yml file already exists. Do you want to write over this file? [y/N]
y
INFO[0002] The network ecs-local-network already exists 
INFO[0002] The amazon-ecs-local-container-endpoints container already exists with ID 67a5669eada760ca766dfde7b90e93c1695e94412691fb420470a3618072b6b1 
INFO[0002] Started container with ID 67a5669eada760ca766dfde7b90e93c1695e94412691fb420470a3618072b6b1 
INFO[0002] Using docker-compose.ecs-local.yml to start containers 
```

File: ecs-cli/modules/cli/local/converter/converter.go [hunk 1]
```diff
@@ -283,6 +283,7 @@ func convertLogging(logConfig *ecs.LogConfiguration) *composeV3.LoggingConfig {
 	if logConfig == nil {
 		return nil
 	}
+	log.Warning("AWS logging will not be supported")
```
Comment (SoManyHs) [inline ecs-cli/modules/cli/local/converter/converter.go]: I would maybe interpolate this to use the `LogDriverAwslogs` constant in the aws-go-sdk.

Comment (iamhopaul123) [reply ecs-cli/modules/cli/local/converter/converter.go]: done!

File: ecs-cli/modules/cli/local/converter/converter.go [hunk 2]
```diff
@@ -283,6 +283,7 @@ func convertLogging(logConfig *ecs.LogConfiguration) *composeV3.LoggingConfig {
 	if logConfig == nil {
 		return nil
 	}
+	log.Warningf("%s will not be supported", ecs.LogDriverAwslogs)
```
Comment (efekarakus) [inline ecs-cli/modules/cli/local/converter/converter.go]: should we only log this if the logConfig driver is set to awslogs ?

Comment (iamhopaul123) [reply ecs-cli/modules/cli/local/converter/converter.go]: done!

**Pull request merged**